Travel Scheduler/View Controllers/HomeCollectionViewController.m

```diff
@@ -174,22 +161,26 @@ - (NSIndexPath *)tableView:(UITableView *)tableView willSelectRowAtIndexPath:(NS
 
 #pragma mark - UICollectionViewDataSource Methods
 
-- (NSInteger)collectionView:(UICollectionView *)collectionView numberOfItemsInSection:(NSInteger)section
-{
+- (NSInteger)collectionView:(UICollectionView *)collectionView numberOfItemsInSection:(NSInteger)section {
     return 5;
 }
 
-- (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView cellForItemAtIndexPath:(NSIndexPath *)indexPath
-{
-    // ----- HEEEEEREEEEE --------
+- (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView cellForItemAtIndexPath:(NSIndexPath *)indexPath {
     [collectionView registerClass:[AttractionCollectionCell class] forCellWithReuseIdentifier:@"AttractionCollectionCell"];
     AttractionCollectionCell *cell = [collectionView dequeueReusableCellWithReuseIdentifier:@"AttractionCollectionCell" forIndexPath:indexPath];
 
     cell.delegate = self;
-    [cell setImage:nil];
+    if (self.areWeInRestaurant) {
```

Comment:
Wouldn't it be easier to call [cell.typeOfPlaces equals @"<attraction/hotel/restuarant>"] (idk the syntax) instead of using self.areWeInRestaurant? Because self.areWeInRestaurant changes every time you dequeue a new cell but the cell itself holds its own property that is guaranteed to be correct and unchanging every time.